test/parallel/test-trace-atomic-deprecation.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
```

Comment:
nit: I suggest using the `node:` prefix here to be consistent with `node:test`
```suggestion
const assert = require('node:assert');
```